components/BottomSheet/examples/BottomSheetWebViewPresentationExample.m

```diff
@@ -0,0 +1,62 @@
+// Copyright 201-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Copyright year.

Reply:
Fixed in b77540002e38b860c978073636d1802647f719ca.